client/components/catalog/Container.vue

```diff
@@ -135,7 +131,7 @@ export default {
     }
   },
   created() {
-    this.fetchTags();
+    this.fetchTags({ addNewTag: false });
```

Comment:
Use something like associated/assigned as flag rather than the current functionality you are adding it for.  It should be falsy by default (no need to specify it).